lib/elixir_analyzer/exercise_test/common_checks/uncommon_errors.ex

```diff
@@ -2,13 +2,14 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.UncommonErrors do
   @moduledoc """
   This is an exercise analyzer extension module used for common tests looking for uncommon errors raised
   """
+  alias ElixirAnalyzer.Constants
 
   defmacro __using__(_opts) do
     quote do
-      feature "raises function clause error" do
+      feature Constants.solution_raise_fn_clause_error() do
```

Comment:
This is a correction that has nothing to do with this PR. We took the convention of common checks names matching the comment so that they can be used to suppress other checks. I missed in in the PR review.